wazimap_ng/general/static/js/variable-filter-widget.js

```diff
@@ -1,15 +1,16 @@
 (function($) {
     jQuery(document).ready(function($) {
         $(function () {
-            $("#variable-permission-filter input[name='variable_type']" ).on('change', ChangeVaribaleValues);
+            $("#variable-permission-filter input[type='radio']" ).on('change', ChangeVaribaleValues);
         });
 
         function ChangeVaribaleValues() {
             let permissionType = $(this).val();
             let hiddenPermissionType = permissionType == "public" ? "private": "public";
-            $("#id_indicator").find("[data-type='"+permissionType+"']").removeClass("hidden");
-            $("#id_indicator").find("[data-type='"+hiddenPermissionType+"']").addClass("hidden");
-            $(document).find("#id_indicator option[value='']").attr('selected', true);
+            let parent = $(this).parents("div");
+            parent.find("#id_indicator").find("[data-type='"+permissionType+"']").removeClass("hidden");
+            parent.find("#id_indicator").find("[data-type='"+hiddenPermissionType+"']").addClass("hidden");
+            parent.find("#id_indicator option[value='']").attr('selected', true);
```

Comment:
please add spaces between the operators, e.g. `find("[data-type='" + permissionType + "']")`

Reply:
just confirmed with JD - he's happy for us to use literal templates for the django admin, so best would be:
```find(`[data-type='${permissionType}']`)```

Reply:
@jbothma Is it ok to use this for admin panel ?

Reply:
Fixed 👍